.github/workflows/test-docker.yml

```
@@ -0,0 +1,9 @@
+name: test-docker
+on: [push]
```

```suggestion

on:
  # Triggers the workflow on push or pull request events but only for the main branch
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]

```

Try this, github would then automatically pickup the action in this PR. :)